src/plugins/timelist/inspector/TimelistPropertiesView.vue

```diff
@@ -78,6 +95,17 @@ const EVENT_TYPES = [
   }
 ];
 
+const DISPLAY_STYLES = [
+  {
+    key: false,
+    label: 'Compact'
+  },
+  {
+    key: true,
+    label: 'Expanded'
+  }
+];
+
```

Comment:
this seems overly complicated

Reply:
How so?

Reply:
We're declaring an array of objects, assigning it to a reactive data property (it gets wrapped in a Proxy), and generating a computed value to solve the problem of hardcoding 2 strings in a ternary statement